package-lock.json

```diff
@@ -1,624 +0,0 @@
-{
```

Comment:
this file should remain committed (see above)

Reply:
done